controllers/util/snapshotclasses.go

```diff
@@ -47,6 +47,9 @@ func NewDefaultRbdSnapshotClass(
 			"clusterID": clusterID,
 			"csi.storage.k8s.io/snapshotter-secret-name":      provisionerSecret,
 			"csi.storage.k8s.io/snapshotter-secret-namespace": namespace,
+
```

Comment:
there is no need to explicitly separate these by a line as all these are just fields in a map?

this isn't required for groupsc?

Reply:
That extra space is to keep git blame cleaner (i can remove it if you want). This change will be reverted in a future release due to upcoming changes in external-snapshotter.

No, this is only for volsnaps.

Reply:
Can remove, the PR isn't huge that warrants a distinction or as it's slated for removal, pls comment the same and you'll get clear diff for free 😅.